core/server/public/members-theme-bindings.js

```diff
@@ -0,0 +1,887 @@
+      hash = $__2[1],
+      query = $__2[2];
+  if (hashMatch && hash === 'reset-password') {
+    var $__3 = query.match(/token=([a-zA-Z0-9-_]+.[a-zA-Z0-9-_]+.[a-zA-Z0-9-_]+)/) || [],
+        tokenMatch = $__3[0],
+        token = $__3[1];
+    if (tokenMatch) {
+      return auth.resetPassword({token: token}).then((function(success) {
+        window.location.hash = '';
+        return success;
+      })).then(reload);
+    }
+  }
+  auth.on('signedin', function() {
+    auth.getSSRToken({fresh: true}).then(function(token) {
+      createSession(token);
+    });
+  });
+  auth.on('signedout', function() {
+    destroySession();
+  });
+  function signout() {
+    auth.signout().then((function() {
+      return destroySession();
+    })).then(reload);
+  }
+  function signin() {
+    auth.signin().then((function() {
+      return auth.getSSRToken({fresh: true}).then(function(token) {
+        return createSession(token);
+      });
+    })).then(reload);
+  }
+  function signup($__4) {
+    var $__6;
+    var $__5 = $__4,
+        coupon = ($__6 = $__5.coupon) === void 0 ? '' : $__6;
+    auth.signup({coupon: coupon}).then((function() {
+      return auth.getSSRToken({fresh: true}).then(function(token) {
+        return createSession(token);
+      });
+    })).then(reload);
+  }
+  function upgrade() {
+    auth.upgrade().then((function() {
+      return auth.getSSRToken({fresh: true}).then(function(token) {
+        return createSession(token);
+      });
+    })).then(reload);
+  }
+  var signinEls = document.querySelectorAll('[data-members-signin]');
+  var signupEls = document.querySelectorAll('[data-members-signup]');
+  var upgradeEls = document.querySelectorAll('[data-members-upgrade]');
+  var signoutEls = document.querySelectorAll('[data-members-signout]');
+  each(signinEls, (function(el) {
+    el.addEventListener('click', (function(event) {
+      event.preventDefault();
+      signin();
+    }));
+  }));
+  each(signupEls, (function(el) {
+    el.addEventListener('click', (function(event) {
+      event.preventDefault();
+      var coupon = el.dataset.membersCoupon;
+      signup({coupon: coupon});
+    }));
+  }));
+  each(upgradeEls, (function(el) {
+    el.addEventListener('click', (function(event) {
+      event.preventDefault();
+      upgrade();
+    }));
+  }));
+  each(signoutEls, (function(el) {
+    el.addEventListener('click', (function(event) {
+      event.preventDefault();
+      signout();
+    }));
+  }));
+}
+function reload(success) {
+  if (success) {
+    window.location.reload();
+  }
+}
+function createSession(token) {
+  return fetch('/members/ssr', {
+    method: 'post',
+    credentials: 'include',
+    body: token
+  }).then(function(res) {
+    return !!res.ok;
+  });
+}
+function destroySession() {
+  return fetch('/members/ssr', {method: 'delete'}).then(function(res) {
+    return !!res.ok;
+  });
+}
+
+//# sourceURL=/home/donny/usr/src/ghost.org/Members/packages/members-theme-bindings/index.js
```

Comment:
`/home/donny/usr/src/` :trollface: Guess its just because it was copied from local build?

Reply:
Lol :joy: Yeah must be - I think it's okay though? Or should we remove?

Reply:
I'm not sure if it will break stuff if removed, i'll give it a go

Reply:
It didn't break stuff, but might make things annoying going forward - I think the correct solution is to force the build to not include sourcemaps - Which we can fix in the Members repo

Reply:
Yeah i think it def has to be to not include sourcemaps in that case later, or it will always include one of our local path which isn't ideal at all even for built stuff 😸